src/main/java/nl/novi/techiteasy/dtos/TelevisionDto.java

```diff
@@ -0,0 +1,173 @@
+package nl.novi.techiteasy.dtos;
+
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+
+@Component
```

Comment:
Een DTO heeft geen architectuur annotatie nodig als `@`Component of `@`Service, omdat een DTO enkel een datadrager klasse is. Het is niet deel van de architectuur (controller, service, repository) en hoeft nergens via injection gebruikt te worden